Makefile

```diff
@@ -276,6 +277,18 @@ test-coverage/%:
 		&& $$CMD ./... \
 		&& $(GO) tool cover -html=coverage.out -o coverage.html;
 
+# Fuzzing
+
+.PHONY: fuzz-config-json
+fuzz-config-json:
+	cd config/v0.3.0 \
+	&& $(GO) test -run=^$ -fuzztime=$(FUZZTIME) -fuzz=FuzzJSON
+
+.PHONY: fuzz-config-yaml
+fuzz-config-yaml:
+	cd config/v0.3.0 \
```

Comment:
How will this scale to multiple versions? Should this be finding related files and then iterating over them instead?

Reply:
I was thinking about it as well but decided to keep it simple before we have scaling issues :wink:

Reply:
Do we need it for only the latest version of config or for all possible fuzz tests?

Reply:
Removed https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6604/commits/4cdabcfd3dac9e2f2aaaf7c8d638d153f22fc37b

1. There is something wrong in the Makefile https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6604/commits/4cdabcfd3dac9e2f2aaaf7c8d638d153f22fc37b (running too long)
2. Fuzzing should be executed for a long time so I do not think it make sense to have it in GitHub Actions anyway - OSS-Fuzz is the way